src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -166,7 +165,19 @@ class ValidationStep extends React.Component {
                             resizeMode="center"
                         />
                         <Text style={[styles.mh5, styles.mb5]}>
-                            {this.props.translate('validationStep.verifyingDescription')}
+                            {this.props.translate('validationStep.reviewingInfo')}
+                            <TextLink
+                                onPress={() => {
+                                    // There are two modals that must be dismissed before we can reveal the Concierge
+                                    // chat underneath these screens
+                                    Navigation.dismissModal();
+                                    Navigation.dismissModal();
```

Comment:
Should we maybe write up some sort of utility function that just dismisses all modals?

Reply:
I actually think it's a good point because I don't like this this double dismiss logic, but I'm not sure I'll get to it right now. 
Would you be willing to create an issue for that so we can look into improving it soon?